test/mitmproxy/tservers.py

```diff
@@ -344,11 +356,15 @@ def setup_class(cls):
         cls.chain = []
         for _ in range(cls.n):
             opts = cls.get_options()
-            tmaster = cls.masterclass(opts)
-            tmaster.addons.add(core.Core())
-            proxy = ProxyThread(tmaster)
+            proxy = ProxyThread(cls.masterclass, opts)
             proxy.start()
             cls.chain.insert(0, proxy)
+            while 1:
```

Comment:
`while True` is a bit more pythonic.

Reply:
Fixed. For extra points, guess what language I've been auditing recently?